CHANGELOG.md

```
@@ -0,0 +1,1107 @@
+  });
+  ```
+
+⚡ Designed to make query execution more transparent, schema management safer, and monitoring easier inside Forge apps.
+
+Мне нужно точно таком же формате только в новая версия вышла
+
+Вот что нового, новый релиз сделан по статью но в релиз ноте не надо указывать про статью. Просто добавлено:
+
+1.  Добавлена Observobility на уровне резолвера
+
+        ```ts
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)